.github/workflows/04-publish-todo-app.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: echo "::set-output name=tag::$(date +'%Y%m%d%H%M%S')-${GITHUB_SHA}"
 
       - name: Publish Docker image to ECR registry
-        if: github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/main' || contains(github.event.head_commit.message, 'DEPLOY NOW')
```

Comment:
I use this technique regularly for other projects to avoid commenting in and out the `if` block if I want to deploy from a branch.

With this change, we can place `DEPLOY NOW` in the commit body when working on a branch and the application gets deployed.

When squashing the commits before merging, we can also ensure to not leave this "garbage" in our commit history.